hack/generate-template-variable-doc.py

```diff
@@ -53,12 +51,10 @@ def update_doc(doc: str, boundary: str, template_path: str) -> str:
     new_doc = re.sub(table_pattern, "\n".join([boundary, *new_table_lines, boundary]), doc)
     return new_doc
 
-
-with open(doc_file_name) as doc_file:
-    doc = doc_file.read()
-
-doc = update_doc(doc, al2_boundary, '../templates/al2/template.json')
-doc = update_doc(doc, al2023_boundary, '../templates/al2023/template.json')
-
-with open(doc_file_name, 'w') as doc_file:
-    doc_file.write(doc)
+for template in ['al2', 'al2023']:
+    doc_file_name = f'../doc/usage/{template}.md'
+    with open(doc_file_name) as doc_file:
+        doc = doc_file.read()
+        doc = update_doc(doc, f'../templates/{template}/template.json')
```

Comment:
while we're in the business of house cleaning for new distros, maybe we can just discover these based on a directory walk. 
just one less thing to keep updating 🤷